torch_xla/csrc/tensor_impl.cpp

```diff
@@ -22,6 +22,10 @@ XLATensorImpl::XLATensorImpl(XLATensor tensor)
   }
 }
 
+XLATensorImpl::XLATensorImpl()
```

Comment:
Why do we need our own "undefined" thing?
It seems we are only creating it to satisfy the "undefined" tensor for the conv2d bias (which is a waste).